client/src/components/Graph/Graph.tsx

```diff
@@ -587,6 +617,286 @@ class Graph extends React.Component<GraphProps, GraphState> {
     if (selectionTool === "lasso") await this.handleLassoDeselectAction();
   }
 
+  handleLidarWheelEvent = (e: WheelEvent): void => {
```

Comment:
theres a lot of inline handlers and code now associated with lidar -worth refactoring into separate module?

Reply:
also, add inline code comments